types/screen.d.ts

```diff
@@ -16,7 +16,7 @@ export type Screen<Q extends Queries = typeof queries> = BoundFunctions<Q> & {
    * Convenience function for `Testing Playground` which logs URL that
    * can be opened in a browser
    */
-  logTestingPlaygroundURL: (element?: Element | HTMLDocument) => void
+  logTestingPlaygroundURL: (element?: Element) => void
```

Comment:
This shouldn't change considering the scope of this PR.

Reply:
It's been a while now, but this change was actually triggered by your suggestion of removing `if (!element || !('innerHTML' in element))` and relying on static type checking: https://github.com/testing-library/dom-testing-library/pull/852#discussion_r555794226

For the static types to be correct after that change, we would have to remove `HTMLDocument` as a valid type, I explained this here: https://github.com/testing-library/dom-testing-library/pull/852#discussion_r557629382.
If I change `getPlaygroundUrl` to also accept `HTMLDocument` it will trigger a compile error:

```
Property 'innerHTML' does not exist on type 'HTMLDocument'.
```

`HTMLDocument` can be kept as part of `logTestingPlaygroundURL`, but it would be invalid and might cause a runtime error when it forwards the argument to `getPlaygroundUrl`.